CHANGELOG-3.5.md

```diff
@@ -227,6 +227,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 - Add [`etcdctl get --count-only`](https://github.com/etcd-io/etcd/pull/11743) flag for output type `fields`.
 - Add [`etcdctl member list -w=json --hex`](https://github.com/etcd-io/etcd/pull/11812) flag to print memberListResponse in hex format json.
 - Changed [`etcdctl lock <lockname> exec-command`](https://github.com/etcd-io/etcd/pull/12829) to return exit code of exec-command.
+- Add support for a config file. Add [`etcdctl --config-file`](https://github.com/etcd-io/etcd/pull/12891) to specify a config file and search in default location in `$HOME/.etcdctl/config.yml`.
```

Comment:
I would suggest a small re-write here. Make it explicit here by saying something like `Add support for providing etcdctl arguments in a config file` instead of `Add support for a config file.` to avoid any confusing with server config file in the first sentence before use read further. Also, it uses provided file or one from default location so may be something like "Add [`etcdctl --config-file`](https://github.com/etcd-io/etcd/pull/12891) flag to specify a config file or use `$HOME/.etcdctl/config.yml` if provided."